src/sentry/coreapi.py

```diff
@@ -399,6 +399,14 @@ def should_filter(self, project, data, ip_address=None):
             if filter_obj.is_enabled() and filter_obj.test(data):
                 return (True, 'other_filter')
 
+        if matches_discarded_hash(data, project):
+            self.log.info(
```

Comment:
Hi.

This is the only `info` call in the entirety of `coreapi`. We talked about this getting downgraded to `debug` in the past, but my suggestion is still to use a `metrics.incr`. Since the only observable part of this recording is the `project_id`, all I see us getting out of this is a "yup, it works.", and if the `project_id` is what we're after, we can just go with:

```
from sentry.utils import metrics
...
metrics.incr('preprocess.discarded', instance=project.id)
```